src/fast_platform.h.template

```diff
@@ -7,7 +7,7 @@
 
 std::string fastnetmon_version = "${FASTNETMON_APPLICATION_VERSION}";
 
-std::string pid_path = "/var/run/fastnetmon.pid";
+std::string pid_path = "";
```

Comment:
I do not like this approach because it changes FNM's behaviour in case when we do not have pid_path specified in configuration explicitly (default behaviour). 

What's the problem with pid path?